dotnet/SimpleSauce/setSauceEnvironmentVariables.ps1

```diff
@@ -0,0 +1,10 @@
+Param(
+[string]$sauceUserName,
+[string]$sauceAccessKey
+)
+Write-Output "sauce.userName that was passed in from Azure DevOps=>$sauceUserName"
+Write-Output "sauce.accessKey that was passed in from Azure DevOps=>$sauceAccessKey"
+Write-Output "Reading sauce.userName directly from Azure DevOps using a different syntax=>" + $env:SAUCE_USERNAME
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Cmdlet 'Write-Output' has positional parameter. Please use named parameters instead of positional parameters when calling a command.](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4471446)